communication/src/main/java/org/jboss/da/communcation/pnc/ReadResource.java

```diff
@@ -0,0 +1,25 @@
+package org.jboss.da.communcation.pnc;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+/**
+ * Created by dcheung on 22/06/15.
+ */
+public class ReadResource {
+
+    public static String getResource(String key) {
```

Comment:
I'll probably remove that file, or rename it to "Configuration". So this class would get PostConstructed with all the properties in it, and then we just have to query it?